static/css/subject_page.scss

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * Styling for disaster dashboard.
+ * Styling for sustainability explorer.
```

Comment:
i think this should be for subject pages?

Reply:
done